src/import/client.rs

```diff
@@ -218,6 +227,34 @@ impl ImportClient for Client {
     fn has_region_id(&self, id: u64) -> Result<bool> {
         Ok(self.pd.get_region_by_id(id).wait()?.is_some())
     }
+
+    fn is_scatter_region_finished(&self, region_id: u64) -> Result<bool> {
+        match self.pd.get_operator(region_id) {
+            Ok(resp) => {
+                // If the current operator of region is not `scatter-region`, we could assume
+                // that `scatter-operator` has finished or timeout.
+                let scatter_region = "scatter-region";
+                if resp.desc.len() != scatter_region.len() || resp.desc != scatter_region.as_bytes()
```

Comment:
```rust
                if resp.desc != scatter_region.as_bytes()
```

The length check is redundant.

Actually perhaps just combined the conditions as

```rust
Ok(resp.desc != scatter_region.as_bytes() || resp.status != OperatorStatus::RUNNING)
```